src/webgpu/shader/validation/statement/return.spec.ts

```diff
@@ -0,0 +1,111 @@
+export const description = `Validation tests for 'return' statements'`;
```

Comment:
This PR has type check tests for return, but no parsing checks for return.  I assume those will come later.

E.g. you can't do `return _ = 1;`

Reply:
Done (added `src/webgpu/shader/validation/parse/return.spec.ts`)